lms/templates/commerce/checkout_receipt.html

```diff
@@ -0,0 +1,57 @@
+<%!
+from django.utils.translation import ugettext as _
+%>
+<%namespace name='static' file='../static_content.html'/>
+
+<%inherit file="../main.html" />
+<%block name="bodyclass">register verification-process step-requirements</%block>
```

Comment:
Would there be any advantage to defining CSS classes and IDs more specific to this page?